### PR TITLE
1388 psss profile photo

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- Add profile image field to PSSS

---

### Screenshots:
See issue https://github.com/beyondessential/tupaia-backlog/issues/1388

### CODE REVIEW

File: packages/psss/src/views/Tabs/ProfileTabView.js [hunk 1]
```diff
@@ -34,38 +36,82 @@ const SuccessMessage = styled(SmallAlert)`
...
+      fileId: `${user.id}-${fileName}`,
+      data: base64,
+    });
+    setStatus(STATUS.IDLE);
+  };
+
+  const handleFileDelete = () => {
+    // We need to set a file id to something so that the backend will pick up the profileImage field
+    // as an updatedField and save the change.
+    setProfileImage({ fileId: '-', data: null });
```
Comment (edmofro) [inline packages/psss/src/views/Tabs/ProfileTabView.js:95]:
```suggestion
    setProfileImage({ fileId: '', data: null });
```
Does that work too?

**Review by edmofro — APPROVED** [review]
LGTM :-)

File: packages/psss/src/views/Tabs/ProfileTabView.js (hunk 1 above)
Comment (tcaiger) [reply packages/psss/src/views/Tabs/ProfileTabView.js:95]: no unfortunately not. In the submit handler, I use the fileId to determine whether or not the profileImage is updated. See line 70. I could tweak that if you think it is confusing.

File: packages/psss/src/views/Tabs/ProfileTabView.js [hunk 2]
```diff
@@ -34,38 +36,82 @@ const SuccessMessage = styled(SmallAlert)`
...
-    setIsLoading(true);
+    setStatus(STATUS.LOADING);
     setErrorMessage(null);
     try {
       await onUpdateProfile({
         first_name: firstName,
         last_name: lastName,
         position: role,
         employer,
+        profileImage: profileImage.fileId && profileImage,
```
Comment (edmofro) [inline packages/psss/src/views/Tabs/ProfileTabView.js:70]: Pointing me to this line made me notice that it's camel case, with the first two fields in snake case

File: packages/psss/src/views/Tabs/ProfileTabView.js (hunk 1 above)
Comment (edmofro) [reply packages/psss/src/views/Tabs/ProfileTabView.js:95]: Yeah I do find it a little confusing - could we check if `profileImage.data !== user.profileImage`?

**Pull request merged**